MAUI/Radial-Menu/Populating-Items.md

```diff
@@ -0,0 +1,723 @@
+               new SfRadialMenuItem() { Text = "Undo", FontSize = 12 },
+               new SfRadialMenuItem() { Text = "Color", FontSize = 12 },
+            };
+            radialMenu.Items = itemCollection;
+            this.Content = radialMenu;
+        }
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+N> Instead of using ObservableCollection, use `RadialMenuItemsCollection` for the RadialMenu Items list and `SubMenuItemsCollection` for the list of items within each RadialMenuItem.
+
+![Display text in .NET MAUI RadialMenu](images/populating-items/maui-radialmenu-populating-only-text.png)
+
+### ImageWithText
+
+The following code snippet demonstrates how to Display images and text in the RadialMenu items of the control.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:local="clr-namespace:RadialSample"
+             xmlns:syncfusion="clr-namespace:Syncfusion.Maui.RadialMenu;assembly=Syncfusion.Maui.RadialMenu"
+             x:Class="RadialSample.MainPage">
+    <syncfusion:SfRadialMenu>
+        <syncfusion:SfRadialMenu.Items>
+            <syncfusion:SfRadialMenuItem Text="John"  Image="johnson.png"/>
+            <syncfusion:SfRadialMenuItem Text="Krish" Image="krish.png"/>
+            <syncfusion:SfRadialMenuItem Text="Ram"   Image="ram.png"/>
+            <syncfusion:SfRadialMenuItem Text="Kather"  Image="kather.png"/>
+            <syncfusion:SfRadialMenuItem Text="Joe"  Image="joe.png"/>
+        </syncfusion:SfRadialMenu.Items>
+    </syncfusion:SfRadialMenu>
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+![Display image with text in .NET MAUI RadialMenu.](images/populating-items/maui-radialmenu-populating-text-with-images.png)
+
+using Syncfusion.Maui.RadialMenu;
+
+namespace RadialSample
+{
+    public partial class MainPage : ContentPage
+    {
+        public MainPage()
+        {
+            InitializeComponent();
+
+            SfRadialMenu radialMenu = new SfRadialMenu();
+            RadialMenuItemsCollection itemCollection = new RadialMenuItemsCollection()
+            {
+                new SfRadialMenuItem() { Text = "John", Image = "johnson.png" },
+                new SfRadialMenuItem() { Text = "Krish", Image = "krish.png" },
+                new SfRadialMenuItem() { Text = "Ram", Image = "ram.png" },
+                new SfRadialMenuItem() { Text = "Kather", Image = "kather.png" },
+                new SfRadialMenuItem() { Text = "Joe", Image = "joe.png" }
+            };
+
+            radialMenu.Items = itemCollection;
+            this.Content = radialMenu;
+        }
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Custom Font with Text
+
+The following code snippet demonstrates how to Display custom font with text in the RadialMenu items of the control.
```

Comment:
The following code sample demonstrates how to Display custom font with text in the RadialMenu items of the control.